site/archive/archiveTypes.ts

```diff
@@ -0,0 +1,12 @@
+export interface UrlAndMaybeDate {
+    url: string
+    date?: Date
```

Comment:
A `previousVersion` and `nextVersion` can have a date attached to it, but that's currently not being rendered. Just some future-proofing.